azkaban-common/src/main/java/azkaban/trigger/builtin/SlaChecker.java

```diff
@@ -162,6 +172,7 @@ public Object isSlaFailed() {
     final ExecutableFlow flow;
     try {
       flow = this.executorLoader.fetchExecutableFlow(this.execId);
+      logger.info("Flow for execid " + this.execId + " is " + flow.toString());
```

Comment:
Why to you need to print out `flow.toString()`? Would `flow.getId()` be better?

Reply:
agreed